src/Symfony/Component/Cache/Tests/Adapter/CouchbaseBucketAdapterTest.php

```diff
@@ -16,7 +16,8 @@
 use Symfony\Component\Cache\Adapter\CouchbaseBucketAdapter;
 
 /**
- * @requires extension couchbase 2.6.0
+ * @requires extension couchbase <3.0.0
```

Comment:
duplicate with check in `setupBeforeClass` because not all version of phpunit supports multiple requires on the same extension